src/components/missions/Missions.css

```diff
@@ -5,8 +5,8 @@
 .not-member,
 .active-member {
   font-weight: bold;
-  background-color: rgb(104, 104, 104);
-  color: white;
+  background-color: rgb(104, 104, 104) !important;
+  color: white !important;
   border: none;
   border-radius: 0.5em;
   font-size: 0.75rem;
```

Comment:
- [ ] It would be great if you use bootstrap classes for styling your buttons. because you have added bootstrap to your project.

Reply:
I removed custom CSS classes and replaced it with react-bootstrap classes.